docs/faq.md

```diff
@@ -351,29 +351,30 @@ If you run experiments with tabulated benchmarks using the `SimulatorBackend`, a
 
 ### <a name="remote-tuning"></a> I don’t want to wait, how can I launch the tuning on a remote machine?
 
-You can use the remote launcher to launch an experiment on a remote machine. The
-remote launcher supports both `LocalBackend` and `SageMakerBackend`. In the
-former case, multiple trials will be evaluated on the remote machine (one use-case
-being to use a beefy machine), in the latter case trials will be evaluated as
-separate SageMaker training jobs.
+Remote launching of experiments has a number of advantages:
+* The machine you are working on is not blocked
+* You can launch many experiments in parallel
+* You can launch experiments with any instance type you like, without having to
+  provision them yourselves
 
-Here is an example for how to run tuning with the remote launcher:
+Remote launching for local, simulator, and SageMaker back-end is detailed in
+[this tutorial](tutorials/benchmarking/README.md), see also these examples:
+* Local backend: [launch_remote.py](../benchmarking/nursery/launch_local/launch_remote.py)
+* Simulator backend: [launch_remote.py](../benchmarking/nursery/benchmark_dehb/launch_remote.py)
+* SageMaker backend: [launch_remote.py](../benchmarking/nursery/launch_sagemaker/launch_remote.py)
+
+* Another way for remote launching is shown in
 [launch_height_sagemaker_remotely.py](../examples/launch_height_sagemaker_remotely.py).
-It may be more convenient for you to write your own launcher scripts. Examples:
-* Local backend: [benchmarking/nursery/launch_local/launch_remote.py](../benchmarking/nursery/launch_local/launch_remote.py)
-* Simulator backend: [benchmarking/nursery/benchmark_dehb/launch_remote.py](../benchmarking/nursery/benchmark_dehb/launch_remote.py)
-* SageMaker backend: [benchmarking/nursery/launch_sagemaker/launch_remote.py](../benchmarking/nursery/launch_sagemaker/launch_remote.py)
 
 ### <a name="experiment-parallel"></a> How can I run many experiments in parallel?
 
-You can call the remote launcher multiple times to schedule a list of experiments.
```

Comment:
This one is more descriptive, and informative of an answer to the question, as compared to the new version.

Reply:
I changed it slightly. As I said, there is no single "remote launcher" anymore.